types/googlemaps/index.d.ts

```diff
@@ -2509,11 +2524,9 @@ declare namespace google.maps {
 
         export class HeatmapLayer extends MVCObject {
             constructor(opts?: HeatmapLayerOptions);
-            getData(): MVCArray;
+            getData<T extends LatLng | WeightedLocation>(): MVCArray<T>;
```

Comment:
Why not have getData not take a parameter and return `MVCArray<LatLng | WeightedLocation>` instead? It would be a lot easier not to require the caller to have to pass in the type.